counsel.el

```diff
@@ -1079,7 +1060,7 @@ INITIAL-INPUT can be given as the initial minibuffer input."
 (defvar counsel-dired-listing-switches "-alh"
   "Switches passed to `ls' for `counsel-cmd-to-dired'.")
 
-(defun counsel-cmd-to-dired (full-cmd &optional process-filter)
+(defun counsel-cmd-to-dired (full-cmd &optional filter)
```

Comment:
Would be nice to split off the straightforward changes like `process-status` from the other changes, much easier to review smaller pieces.

Reply:
OK.